customerio-reactnative-richpush.podspec

```diff
@@ -0,0 +1,25 @@
+require "json"
+
+package = JSON.parse(File.read(File.join(__dir__, "package.json")))
+
+Pod::Spec.new do |s|
+  s.name         = "customerio-reactnative-richpush"
```

Comment:
We need to create a separate `richpush` CocoaPod because for the NSE target, we only want the native iOS SDKs to install. The `customerio-reactnative.podspec` file also installs the in-app SDK and React Native dependencies which we don't want in a NSE target.

Reply:
hmm if `customerio-reactnative.podspec` already has the same subspec logic, 

```
  s.subspec 'apn' do |ss|
    ss.dependency "CustomerIO/MessagingPushAPN", package["cioNativeiOSSdkVersion"]
  end

  s.subspec 'fcm' do |ss|
    ss.dependency "CustomerIO/MessagingPushFCM", package["cioNativeiOSSdkVersion"]
```

wouldn't that mean,  if the apn/fcm subspec is used, only then following dependency will be installed? 

can't we just refer to that from apps podfile?

Reply:
Good question. I wondered the same thing, tried it, and found it wasn't possible. 

I [added docs](https://github.com/customerio/customerio-reactnative/pull/149/commits/bd060fcdba4d0f764d90480e584c27d9f4b25049) explaining why this isn't possible. If those new comments do not help explain why to you, add a comment here to this thread and I'll make sure to update the docs to be more helpful.